src/common/components/icons/collection-icon.component.tsx

```diff
@@ -0,0 +1,15 @@
+export const TableIcon = () => {
+  return (
+    <svg
+      xmlns="http://www.w3.org/2000/svg"
+      width="1.2em"
+      height="1.2em"
```

Comment:
Perhaps for better visibility, we can increase its size to 1.3em, as it appears slightly smaller than the other icons.